.github/release-drafter.yml

```
@@ -1,19 +1,176 @@
+      # Test-only changes
+      - 'zebra-test'
+      - '.cargo/config.toml'
+      - 'clippy.toml'
+      # CI-only changes
+      - '.github'
+      - '.codespellrc'
+      - 'codecov.yml'
+      - 'deny.toml'
+
+# The release name, tag, and settings for the draft CHANGELOG.
+name-template: 'Zebra $RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+tag-prefix: 'v'
 prerelease: true
+
+# Categories in rough order of importance to users.
+# Based on https://keepachangelog.com/en/1.0.0/
+category-template: '### $TITLE'
 categories:
-  - title: 'Features'
-    labels:
-      - 'feature'
-      - 'enhancement'
-  - title: 'Bug Fixes'
-    labels:
-      - 'fix'
-      - 'bugfix'
-      - 'bug'
-  - title: 'Maintenance'
-    label: 'chore'
-change-template: '- $TITLE (#$NUMBER)'
-change-title-escapes: '\<*_&' # You can add # and @ to disable mentions, and add ` to disable code blocks.
-template: |
-  ## Changes
+  - title: 'Security'
+    labels:
+      - 'C-security'
+      # Other labels that are usually security issues
+      - 'I-bad-code'
+      - 'I-bad-data'
+      - 'I-consensus'
+      - 'I-crash'
+      - 'I-destructive'
+      - 'I-hang'
+      - 'I-lose-funds'
+      - 'I-privacy'
+      - 'I-remote-node-overload'
+      - 'I-unbounded-growth'
+      - 'I-unsound'
+  - title: 'Removed'
+    labels:
+      - 'C-removal'
+      # TODO: add breaking change label here
+  - title: 'Deprecated'
+    labels:
+      - 'C-deprecation'
+      These features might be removed in Zebra $NEXT_MINOR_VERSION
+  - title: 'Added'
+    labels:
+      - 'C-feature'
+  - title: 'Changed'
+    labels:
+      - 'C-enhancement'
+  - title: 'Fixed'  
+    labels:
+      - 'C-bug'
+      # Other labels that are usually bugs
+      - 'I-build-fail'
+      - 'I-integration-fail'
+      - 'I-panic'
+  # TODO: if we're happy with the trivial PRs, use "exclude-labels:" instead
+  - title: 'Trivial *TODO:* put this in a PR comment, not the CHANGELOG'
+    labels:
+      # TODO: add trivial label here
+      - 'C-cleanup'
+
+# The next release's $RESOLVED_VERSION, based on the labels of the PRs in the release.
+#
+# In Zebra, we use major versions for mainnet network upgrades,
+# and minor versions for less significant breaking changes.
+version-resolver:
+  major:
+    labels:
+      # We increment the major release version manually
+  minor:
+    labels:
+      - 'C-feature'
+      # TODO: add breaking change label here
+      - 'C-removal'
+      - 'C-deprecation'
+  # We increment the patch version for every release
+  default: patch
+
+# How PR names get turned into CHANGELOG entries.
+change-template: '- $TITLE ([#$NUMBER]($URL))'
+sort-by: title
+sort-direction: ascending
+# Characters escaped when converting PR titles to CHANGELOG entries.
+# Add ` to disable code blocks.
+change-title-escapes: '\<*_&#@'
+# Strip PR series numbers, leading spaces, and conventional commit prefixes from PR titles.
+replacers:
+  - search: '/- [0-9\. ]*([a-zA-Z0-9\(\)]+:)?/'
```

We're missing the exclamation mark.
Maybe something like this?
```suggestion
  - search: '/- [0-9\. ]*([a-zA-Z0-9\(\)!]+:)?/'
```

---

Good catch, thanks!